packages/cashier-v2/src/components/DummyRoute/DummyRoute.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui';
+import { TRouteComponent } from '../../constants/routesConfig';
+
+const DummyRoute = ({ title }: TRouteComponent) => {
```

Comment:
DummyComponent? it did confused my brain with Route

Reply:
sorted